packages/ses/README.md

```diff
@@ -27,34 +27,79 @@ npm install ses
 
 ## Usage
 
-### Module
+### Lockdown
 
-This example locks down the current realm, turning it into a starting
-compartment.
-Within a compartment, there is a `Compartment` constructor that conveys
-"endownments" into the new compartment's global scope, and a `harden` method
-that that object and any object reachable from its surface.
-The compartment can import modules and evaluate programs.
+SES introduces the `lockdown` function.
+Alters the surrounding execution environment such that no object intrinsically
+accessible, like the array or function prototypes (`[].prototype` or `(() =>
+0).prototype`, can be replaced or subverted to attack other code in the same
+execution environment; and tames other intrinsically accessible utilities like
+regular expressions, random numbers, and clocks, to minimize opportunities for
+malicious programs to covertly infer the behavior of other programs running in
+the same execution environment.
 
 ```js
-import {lockdown} from "ses";
+import 'ses';
+
+import 'my-vetted-shim';
+
+lockdown();
```

Comment:
Hm, that shows off the simple one-file use case well. I'm on the fence as to whether `my-vetted-shim` should be imported before `ses` (to visually suggest that it is unconfined), or after (to remind readers that nothing is actually confined until `lockdown()` is called).

But a more realistic usage summary would involve confined modules too. With the current feature set, that means writing an `install-ses` module (consisting entirely of `import 'ses'; lockdown();`), and doing the other imports before/after it:

```js
import 'my-vetted-shim';
import 'install-ses';
import 'confined-module';
other_confined_code();
```
Do we want to show this here/now, or will the upcoming module-import support provide a better way? I'm hoping we'll have some docs that explain how to use this in a way similar to how folks are used to writing/running programs today, rather than putting off the practicalities in favor of a future application-loader/manifest/"larger-scale" ecosystem that we're still trying to figure out.

Reply:
`my-vetted-shim` is ***not*** confined; which is why it, along with any code run before `lockdown()`, must be vetted. IOW, all code run before `lockdown()` can do powerful dangerous things, creating unlimited damage to everything run afterward, including `lockdown()` itself.

Running before `import 'ses';` vs between it and `lockdown()` makes only one difference: whether `harden`,`ModuleStaticRecord`,`Compartment`, and `lockdown` are available as globals. 

I just looked at the shim source and confirmed that `import 'ses';` by itself causes *no* effects other than making these available as properties on the global object and thereby as global variables.

Reply:
I believe @warner is speaking to the point of an `install-ses` module that looks like:

```js
import 'ses';
lockdown();
```

Such that in his example, `confined-module` is running after lockdown.

My intuition is that this is dangerously clever since the so-called confined module still has access to power that lockdown was unable to expunge from the start compartment.

Not sure whether we should highlight that usage pattern, as I am uncertain about how dangerous it is.